docs/content/en/schemas/v2alpha3.json

```diff
@@ -1606,7 +1606,7 @@
           "type": "array",
           "description": "path to Kustomization files.",
           "x-intellij-html-description": "path to Kustomization files.",
-          "default": "[\".\"]"
```

Comment:
this change caused #3618
https://github.com/GoogleContainerTools/skaffold/pull/3620/files

Reply:
@dgageot should this be removed?